### PR TITLE
Added service Service.js: generic CRUD service

### LINKED ISSUE #10: Criar serviço de ações básico.
Serviço de ações básico para gerenciar rota, methods, headers e body.

### PR DESCRIPTION
**Pull request by agueths**
fixed #10

### CODE REVIEW

File: frontend/src/services/Service.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import Fetcher from "../helpers/Fetcher";
+
+/**
+ * Default services (aka C.R.U.D.)
+ */
+class Service
+{
+    /**
```
Comment (paulopieczarka) [inline frontend/src/services/Service.js]: A rota dos params pode ser protected na classe.
route: String;

e no construtor:
constructor(route = "/") {
       this.route = route;
}

File: frontend/src/services/Service.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+     * Create a new object
+     * @param {service} route 
+     * @param {form params} values 
+     */
+    create(route, values)
```
Comment (paulopieczarka) [inline frontend/src/services/Service.js]: remover`route, `

File: frontend/src/services/Service.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            .then(response => console.log(result))
+            .catch(error => console.error(error));
+    }
+
+    /**
+     * List all objects. Can receive a list of filters fields
+     * @param {service} route 
+     * @param {array} fields 
+     */
+    index(route, fields)
```
Comment (paulopieczarka) [inline frontend/src/services/Service.js]: remover`route, `

File: frontend/src/services/Service.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            .then(response => console.log(response))
+            .catch(err => console.log(err));
+    }
+
+    /**
+     * Get a single object.
+     * @param {service} route 
+     * @param {UUID} uid 
+     */
+    get(route, uid)
```
Comment (paulopieczarka) [inline frontend/src/services/Service.js]: remover`route, `

File: frontend/src/services/Service.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+            .catch(err => console.log(err));
+    }
+
+    /**
+     * Update a specific object
+     * @param {service} route 
+     * @param {UUID} uid 
+     * @param {form params} values 
+     */
+    update(route, uid, values)
```
Comment (paulopieczarka) [inline frontend/src/services/Service.js]: remover`route, `

File: frontend/src/services/Service.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+            .then(response => console.log(result))
+            .catch(error => console.error(error));
+    }
+
+    /**
+     * Destroy an object
+     * @param {service} route 
+     * @param {UUID} uid 
+     */
+    delete(route, uid)
```
Comment (paulopieczarka) [inline frontend/src/services/Service.js]: remover`route, `

File: frontend/src/services/Service.js (hunk 2 above)
Comment (agueths) [reply frontend/src/services/Service.js]: :+1:

File: frontend/src/services/Service.js (hunk 3 above)
Comment (agueths) [reply frontend/src/services/Service.js]: :+1:

File: frontend/src/services/Service.js (hunk 4 above)
Comment (agueths) [reply frontend/src/services/Service.js]: :+1:

File: frontend/src/services/Service.js (hunk 5 above)
Comment (agueths) [reply frontend/src/services/Service.js]: :+1:

File: frontend/src/services/Service.js (hunk 6 above)
Comment (agueths) [reply frontend/src/services/Service.js]: :+1:

File: frontend/src/services/Service.js (hunk 1 above)
Comment (agueths) [reply frontend/src/services/Service.js]: fixed :+1:

File: frontend/src/services/Service.js (hunk 6 above)
Comment (paulopieczarka) [reply frontend/src/services/Service.js]: remover `route,`

**Pull request merged**